Sources/SourceKitLSP/Swift/SwiftLanguageServer.swift

```diff
@@ -119,6 +119,11 @@ public final class SwiftLanguageServer: ToolchainLanguageServer {
 
   var commandsByFile: [DocumentURI: SwiftCompileCommand] = [:]
 
+  var documentParseTransition: [DocumentURI: LookaheadRanges] = [:]
```

Comment:
The variable is now incorrectly named. It doesn’t contain any parse transitions.